taxi/tests/test_models.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+class TestModels(TestCase):
+    def test_manufacturer_str(self) -> None:
+        manu = Manufacturer.objects.create(name="Citroen",
+                                           country="France")
+        self.assertEqual(str(manu), f"{manu.name} {manu.country}")
+
+    def test_driver_str(self) -> None:
+        driver = Driver.objects.create(username="testi",
+                                       password="pastest",
+                                       first_name="Test",
+                                       last_name="Testovich")
+        self.assertEqual(str(driver), f"{driver.username}"
+                                      f" ({driver.first_name}"

```

Comment:
It`s better to indent each argument, like this:
![image](https://user-images.githubusercontent.com/123254785/228299888-8249dc74-4135-4725-a818-bfc38ea6fc28.png)